assessment.rb

```diff
@@ -1,12 +1,65 @@
 require 'csv'
 # Assessment answers to:
+class CreateLicensees < ActiveRecord::Migration
+  def change
+    create_table :licensees do |t|
+      t.string :surname
+      t.string :given_name
+      t.string :gender
+      t.integer :height
+      t.integer :weight
+      t.date :dob
+      t.string :eye_color
+      t.string :hair_color
+
+      t.timestamps null: false
+    end
+  end
+end
+
 
 # question 2:
+bishop = Person.create surname: 'Bishop', given_name: 'Caren', gender: 'f', height: 63, weight: 132, dob: '1943-09-26', eye_color: Brown, hair_color: 'Black'
 
 # question 3:
+require 'csv'
+namespace :db do
+  namespace :populate do
+    desc 'Fill the licensees table with example data'
+    task licensees: :environment do
+      City.transaction do
+        #pass in the file that has the data we want to load
+        CSV.foreach(Rails.root + 'data/licensees.csv', headers: true) do |licensee_row|
+          licensee = licensee_row.to_hash
+          next if Licensee.exists? licensee
+          Licensee.create! licensee
+        end
+      end
+    end
 
 # question 4:
 
+# Use ActiveRecord to retrieve all the licensees with Hazel eye color and Brown or Black hair color.
+
+licensees = Licensee.find(:eye_color => 'Hazel' AND :hair_color => 'Brown' OR :eye_color = >'Hazel' AND :hair_color => 'Black')
```

Comment:
No, sorry, this syntax isn't correct. `.find` doesn't take a conditional, it takes a _query_ (in the form of either a string, an array, or a hash).